### PR TITLE
[Solved #50] Establish socket communication

### PR DESCRIPTION
**Pull request by supunTE**
- Establish socket communication
- Add functionality to clear socket-id, and token map when a player disconnected.

### CODE REVIEW

File: apps/server/src/stores/token-player-map.ts [hunk 1]
```diff
@@ -31,4 +31,43 @@ export class TokenPlayerMap {
       this.map.set(authToken, { ...data, socketId });
     }
   }
+
+  //get player id by socket id
```
Comment (dalanad) [inline apps/server/src/stores/token-player-map.ts]: Let's use jsdoc type comments to describe functions

**Review by dalanad — CHANGES_REQUESTED** [review]

File: apps/server/src/stores/token-player-map.ts [hunk 2]
```diff
@@ -24,11 +28,66 @@ export class TokenPlayerMap {
...
+      if (value.socketId === socketId) {
+        return key;
+      }
+    }
+    return null;
+  }
+
+  viewMap(): void {
+    console.log(this.map);
+  }
```
Comment (supunTE) [inline apps/server/src/stores/token-player-map.ts:85]: This function changed in the newest PRs.
28bc2e9: feat(server): make store state and token player map viewable from the server as a static

File: apps/server/src/stores/token-player-map.ts (hunk 1 above)
Comment (supunTE) [reply apps/server/src/stores/token-player-map.ts]: Docs updated 7c8500527f286823f1793b8cdd7cdb83b60b7268

**Review by dalanad — APPROVED** [review]

**Comment by supunTE** [comment]
Solved #50

**Pull request merged**